sunrisemvmtnyc/legislation-tracker
# design: Update main page

resolves #83 

# To test:
Visit page, compare to [figma](https://www.figma.com/design/9iFKBdvUweHd2PKz1XviTc/Sunrise-Legislative-Tracker-Design?node-id=962-538&node-type=canvas&t=romDypZn6HSoimpu-0) and issue #83. 
Check filters & make sure they work.
Verify mobile behavior.

some notes:
- 650px seems too large for the banner. 
- don't actually have any fonts on page
- dropped the gap & widths on the filters
- didn't add card/line toggle - no design for line
- "sort by" - sort by what?
- "take action" - what does this do...?
- didn't bother w/ "district" in homepage bill card
